.github/workflows/shared-publish-java-to-docker-versioned.yaml

```diff
@@ -112,15 +107,15 @@ jobs:
           new_version=${{ steps.version.outputs.new_version }} 
           sed -i "0,/$current_version/s/$current_version/$new_version/" pom.xml
           echo "Version number updated from $current_version to $new_version"
-          echo "image_tag=${{ steps.version.outputs.new_version }}-${{ inputs.cloud_provider }}" >> $GITHUB_OUTPUT
```

Comment:
Not 100% sure if it's related, but when we validate the image here: https://github.com/IABTechLab/uid2-shared-actions/blob/main/.github/workflows/shared-validate-image.yaml#L85 we have the docker tag with `cloud_provider`. If we removed `cloud_provider` here, maybe we should change it accordingly in the validator-image workflow. But how can we distinguish which provider is it?

Reply:
We do actually use the shared-validate-image with different cloud providers from the operator repo, so that should remain. But most repos are calling it 3 times and they don't need to - they just need to use 'default'.